### PR TITLE
Organize playbook with includes.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

File: playbook.yml [hunk 1]
```diff
@@ -1,158 +1,23 @@
...
+# Start of the playbook after the minus sign
+-
+  pre_tasks:
+    - hosts: multi
+      sudo: yes
+      tasks:
+      - name: Add mesosphere deb repository keys
+        apt_key: keyserver=keyserver.ubuntu.com id=E56151BF
 
-    - name: Setup hostname for mesos-master
```
Comment (johngian) [inline playbook.yml:64]: Please fix this (already existing) typo to: 
`Setup ip for mesos-master`

**Comment by johngian** [comment]
Provisioned it locally and seems to work fine:
- Mesos master has 3 slaves
- Marathon gets started
- Zookeeper works

**Comment by akatsoulas** [comment]
@johngian pr updated

**Comment by johngian** [comment]
r+

**Pull request merged**